spec/models/category_spec.rb

```diff
@@ -40,7 +40,7 @@
     context 'for psid-source Category' do
       let(:category) { create(:category, name: 9964305, source: 'psid') }
       let(:course) { create(:course) }
-      let!(:article) { create(:article, title: 'A cappella') }
+      let!(:article) { create(:article, title: 'A cappella', id: 2411) }
```

Comment:
I added that because, I saw that here it was taking id as `100001974`, whereas the id is `2411`
I saw in `PetScan result`, the id for the article `A cappella` is `2411`

Reply:
The id of an Article record won't match the id from petscan. There's a different id in Article for the mediawiki page id.

Reply:
So how to test in that case?

Reply:
For testing this, I wouldn't worry about Article records at all. The way you can find out if feature worked is to check for the expected title in the Category's `article_titles`. If it's there, then if/when the corresponding `A

Reply:
Okay, I got it that, have to `wiki` from `PetScan` and just use the `Article.where(namespace: 0, wiki_id: wiki_id, title: article_titles).pluck(:id)`

But I'm not sure how to get the `Wiki_id` from the `Wiki`

Reply:
If you're going the route we discussed of just warning people that they need to use petscan with the right wiki, then you won't need to worry about extracting the wiki from petscan data.